src/v2/spec.yaml

```diff
@@ -6806,6 +6835,45 @@ paths:
           $ref: "#/components/responses/400Response"
         "500":
           $ref: "#/components/responses/500Response"
+  /farcaster/frame/notification_tokens:
+    get:
+      tags:
+        - Frame
+      summary: |
+        List of frame notification tokens.
+      description: |
+        Returns a list of notifications tokens related for an app
+      externalDocs:
+        url: https://docs.neynar.com/reference/list-notification-tokens
+      operationId: list-notification-tokens
+      parameters:
+        - name: limit
+          in: query
+          description: Number of results to fetch
+          required: false
+          example: 30
+          schema:
+            type: integer
+            default: 20
+            minimum: 1
+            maximum: 100
+          x-is-limit-param: true
+        - name: fids
```

Comment:
Please add [x-comma-separated](https://github.com/neynarxyz/OAS/blob/main/contribution.md#5-handle-comma-separated-values-with-x-comma-separated) and [x-accept-as](https://github.com/neynarxyz/OAS/blob/main/contribution.md#6-use-x-accept-as-for-type-conversion)
[example](https://github.com/neynarxyz/OAS/blob/7827f8d5663a6bceb08bc593fd06c86ab211e87c/src/v2/spec.yaml#L4705)

This will make sure that the generated code in the SDK will accept fids as an array of numbers rather than a comma separated string